notebook/Solution/04 - LSTM-Multivariate-Input-Univariate-Output-Single-Step.ipynb

```diff
@@ -0,0 +1,3205 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": false,
+    "editable": false
+   },
+   "source": [
+    "> **Copyright &copy; 2020 CertifAI Sdn. Bhd.**<br>\n",
+    " **Copyright &copy; 2021 CertifAI Sdn. Bhd.**<br>\n",
+    " <br>\n",
+    "This program and the accompanying materials are made available under the\n",
+    "terms of the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0). \\\n",
+    "Unless required by applicable law or agreed to in writing, software\n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS, WITHOUT\n",
+    "WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the\n",
+    "License for the specific language governing permissions and limitations\n",
+    "under the License. <br>\n",
+    "<br>**SPDX-License-Identifier: Apache-2.0**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multivariate Input ,Univariate Output LSTM ,Single Step\n",
+    "\n",
+    "## Introduction\n",
+    "This tutorial is to demonstrate the process to build LSTM for the Multivariate Input,Univariate Output LSTM ,Single Step<br>\n",
+    "It consists of 2 major part which are :<br>\n",
+    "\n",
+    "Part 1 - The Demostration of Building LSTM using Synthetic data<br>\n",
+    "Part 2 - Exercise of building LSTM using COVID-19 Data.<br>\n",
+    "\n",
+    "\n",
+    "## What will we accomplish?\n",
+    "First, we will show the step in building the LSTM :\n",
+    "\n",
+    "Step 1. Data Preparation (Data Splitting, Data Sequencing, Data Standardization and Batching the Data)<br>\n",
+    "Step 2. Model Configuration (Vanila LSTM , Stacked LSTM , Bidirectional LSTM)<br>\n",
+    "Step 3. Train the model<br>\n",
+    "Step 4. Validate the model using a graph<br>\n",
+    "Step 5. Evaluation Metrics such as MSE<br>\n",
+    "Step 6. Plot the forecast result<br>\n",
+    "\n",
+    "Secondly, learn to import the function from module `data_module.py` and `deep_learning_module.py`<br>\n",
+    "\n",
+    "Thirdly, some data exploration technique on COVID-19 data\n",
+    "\n",
+    "First , let's us import the package needed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "# pip install this package to view the summary of LSTM due to original torchsummary does have the function to view the LSTM\n",
+    "# used jupyter install due to it does not have conda version\n",
+    "# %%capture suppress information of torchsummaryX installation\n",
+    "!pip install torchsummaryX"
```

Comment:
suggest to include torchsummaryX to environment

Reply:
![image](https://user-images.githubusercontent.com/59526258/111237841-de3ca280-8630-11eb-951c-f134a8b3c63d.png)
cannot found torchsummaryX in conda environment, hence use pip install